### PR TITLE
[Finishes #105433088] Changes to randomNumber

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Get rid of `randomNumber(max)` though existing programs that use it should still work.

Add Math.random block
image

randomNumber should show up in autocomplete, since we still have the randomNumber(min, max) version
image

randomNumber intellisense should show this version
image

### CODE REVIEW

File: apps/src/dropletUtils.js [hunk 1]
```diff
@@ -275,9 +277,9 @@ exports.generateDropletPalette = function (codeFunctions, dropletConfig) {
     var blockPair = {
       block: block,
       expansion: expansion,
-      title: cf.func
+      title: funcInfo.func
     };
-    mergedCategories[cf.category].blocks.push(blockPair);
+    mergedCategories[funcInfo.category].blocks.push(blockPair);
```
Comment (Bjvanminnen) [inline apps/src/dropletUtils.js:282]: This block of changes is not strictly necessary, and should be a no-op, but I don't like seeing variables like "cf"

File: apps/i18n/common/en_us.json [hunk 2]
```diff
@@ -113,10 +113,12 @@
   "dropletBlock_mathMax_param0": "n1, n2,..., nX",
   "dropletBlock_mathMax_param0_description": "One or more numbers to compare.",
   "dropletBlock_mathMax_signatureOverride": "Math.max(n1, n2, ..., nX)",
-  "dropletBlock_mathMin_description": "Takes the minimum value among one or more values n1, n2, ..., nX",
+  "dropletBlock__description": "Takes the minimum value among one or more values n1, n2, ..., nX",
```
Comment (joshlory) [inline apps/i18n/common/en_us.json]: How does this value get used?

Comment (Bjvanminnen) [inline apps/i18n/common/en_us.json]: It gets used to populate the blue tooltips

**Comment by joshlory** [comment]
LGTM after changing `dropletBlock__description` -> `dropletBlock_randomNumber_description`.

**Pull request merged**